x-pack/plugins/rule_registry/server/alert_data_client/alerts_client_factory.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient, KibanaRequest, Logger } from 'src/core/server';
+import { PublicMethodsOf } from '@kbn/utility-types';
+import { SecurityPluginSetup } from '../../../security/server';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { AlertingAuthorization } from '../../../alerting/server/authorization';
```

Comment:
Should we make it so the alerting plugin exports this from its root instead of disabling the ESLint check?